File: 03-Booleans And Conditions/booleanOperators.js
function operators(){
    const x = 5;
    const y = 7;

    console.log(x == y);
    console.log(x != y);
    console.log(x > y);
    console.log(7 >= 7);
    console.log(x < y);
    console.log(x <= y);

    console.log('--------');

    console.log(x == y && y === y);
    console.log(x > 2 && y > x);
    console.log(x >= y || x > 5);
    console.log(x >= y || x >= 5);
    console.log(!(x >= y) || x > 5);
    
    
}

operators();

File: 03-Booleans And Conditions/BooleanOperators.java
public class BooleanOperators {
    public static void main(String[] args) throws Exception {
        operators();

    }

    public static void operators(){
        int x = 5;
        int y = 7;

        System.out.println(5 == 7);
        System.out.println(5 != 7);
        System.out.println(x > y);
        System.out.println(x >= y);
        System.out.println(x < y);
        System.out.println(7 <= 7);
        

        System.out.println("----------");
        System.out.println(x == 5 && x < y && x > 7 && y <= x ||);
        System.out.println(x < 6 || y > x);
        System.out.println(!(x < 6) || !(y > x));
        System.out.println(!(x < 6 || y > x));




    }
}
